scripts/run-integration-tests-npm.sh

```diff
@@ -297,7 +297,7 @@ fi
 
 # --- Running Mock Blockchain --- {{{
 log "Running hardhat local node"
-HOPR_ENVIRONMENT_ID="hardhat-localhost" yarn workspace @hoprnet/hopr-ethereum hardhat node \
+HOPR_ENVIRONMENT_ID="hardhat-localhost" yarn workspace @hoprnet/hopr-ethereum exec "hardhat node" \
```

Comment:
Why is this needed? Has worked before afaict.

Reply:
For me, it fails with `no script named hardhat found` - adding exec makes it clear that yarn is executing a script that is  not part of package.json